samples/attested_tls/client/enc/CMakeLists.txt

```diff
@@ -18,7 +18,8 @@ add_executable(tls_client_enc
 	       cert_verifier.cpp
 	       identity_verifier.cpp
 	       ../../common/utility.cpp
-	       ${CMAKE_CURRENT_BINARY_DIR}/tls_client_t.c)
+         ${CMAKE_CURRENT_BINARY_DIR}/tls_client_t.c)
```

Comment:
nit: could we realign the dependencies and add a line between them for readability reasons, please.